dart-mode.el

```diff
@@ -839,6 +862,40 @@ true for positions before the start of the statement, but on its line."
          ((?{) (dart-in-block-p (c-guess-basic-syntax))))))))
 
 
+(defun dart--process-nav-info (response offset)
+  "Report the possible completions and jump to the location of the definition.
+
+  Opens a new file in a new buffer if necessary."
```

Comment:
Explain what the parameters are in the docstring.

Also, the second line should not be indented. You can run `checkdoc-defun` to lint your docstrings against standard Elisp style.